src/Loader/CucumberNDJsonAstLoader.php

```diff
@@ -102,7 +109,7 @@ private static function getFeature(array $json, string $filePath): ?FeatureNode
     private static function getTags(array $json): array
     {
         return array_map(
-            static fn (array $tag) => preg_replace('/^@/', '', $tag['name']),
+            static fn (array $tag) => (string) preg_replace('/^@/', '', $tag['name']),
```

Comment:
(PHPStan) not sure in which situation, but PHPStan thinks preg_replace might return null (perhaps in case of error?)